eng/pipelines/jobs/test-binaries.yml

```diff
@@ -66,7 +71,32 @@ jobs:
         artifactName: Build_Binaries_${{ parameters.configuration }}_${{ parameters.osGroup }}_${{ parameters.architecture }}
         targetPath: '$(Build.SourcesDirectory)/artifacts'
 
-    - ${{ if ne(parameters.useHelix, 'true')}}:
+    - ${{ if eq(parameters.useHelix, 'true')}}:
```

Comment:
Should all of this be skipped for musl-libc since official nodejs only runs on glibc?